docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,64 @@
+Cluster and Services Configurations
+===================================
+
+This section contains information describing the list of available Cluster and
+Services Configurations, the :term:`Namespace` to which they belong,
+the :term:`ConfigMap` where these configurations are store and finally the
+different ways of customizing and applying any given Cluster and Services
+Configurations.
+
+Managing Cluster and Services Configurations
+********************************************
+
+Newly deployed **MetalK8s** clusters come with chosen default values for most
+cluster services. These default values are totally transparent to Admin users
+and can be customized at any point in time given that Admins follow the
+documented procedure to the later.
+
+.. note::
+
+   Cluster and Services Configurations remain persisted throughout the life
+   time of a deployed MetalK8s cluster even after operations like Node reboots,
+   cluster upgrade or downgrade scenarios.
```

Comment:
cluster upgrade(s?), or

Reply:
I think the `scenarios` in there puts `cluster upgrade and downgrade`  in plural form no??

Reply:
Aaaah ok yes, I get it sorry :D